docs/ARDUPILOT.md

```diff
@@ -0,0 +1,55 @@
+# mLRS Documentation: ArduPilot Setup #
+
+([back to main page](../README.md))
+
+([back to CRSF page](CRSF.md))
+
+Further parameters can be configured to optimize peformance for ArduPilot and the RF mode (19Hz, 31Hz or 50Hz) selected.
+
+## mLRS Rx Module Setup
+
+- Rx Snd RadioStat= ardu_1
+
+## Serial Port Setup
```

Comment:
this piece IMHO should be in CRSF.
here I would expect more explanations, like e.g. how the baudrate affects param upload etc

Reply:
Added 'ardu_1' to CRSF page.